src/precice/SolverInterface.hpp

```diff
@@ -772,6 +772,47 @@ class SolverInterface {
       int     valueIndex,
       double &value) const;
 
+  /**
+   * @brief setBoundingBox Define a region of interest in order to filter a
+   *        received mesh for a certain mesh region
+   *
+   * @param[in] dataID ID of the Data you want to access through the bounding box
+   * @param[in] boundingBox (axis aligned) which has in 3D the format
+   *            [x_min, x_max, y_min, y_max, z_min, z_max]
+   *
+   * @return ID of the bounding box (similar to a mesh ID)
+   */
+  int setBoundingBox(
+      const int     dataID,
+      const double *boundingBox) const;
+
+  /**
+   * @brief getLocallyRelevantMeshSize
+   *
+   * @param[in] dataID of the associated mesh. Most probably the data ID of your local readData
+   *
+   * @return Size of the locally relevant mesh as given due to the bounding box
+   *
+   * @pre bounding box has been defined using @p setBoundingBox
+   */
+  int getLocallyRelevantMeshSize(const int dataID) const;
```

Comment:
This is exactly `getMeshVertexSize`, right? I don't think we need an additional API function or am I missing sth?

Reply:
No, this works out using the current API function.